tutorials/nextcloud-instantapp/index.mdx

```diff
@@ -22,49 +24,77 @@ In this tutorial we show you how to can deploy your personal cloud storage with
 
-1. Copy the IP address and open an [SSH session](/compute/instances/how-to/create-an-instance) to find the database credentials required for NextCloud.
+    Go to http://01e090cd-e0f4-4b5e-b035-5fda387535ed.pub.instances.scw.cloud/nextcloud, fill the form with your database information
+    and your desired user.
 
-    The MySQL password appears in the MOTD message that displays upon your first connection to the server:
+    You can hide this message for the next connections by deleting the /etc/update-motd.d/70-nextcloud file.
+    #################################################
+    ```
+3. Open a web browser and enter the NextCloud URL from the MOTD in the navigation bar. NextCloud's installation script displays.
+4. Enter a username and password (of your choice) as well as the database credentials shown in the MOTD of your Instance. Leave the database host as `localhost`. Then click **Install**.
+    <Lightbox src="scaleway-instantapp-nextcloud-install.webp" alt="" />
+    <Message type="note">
+      The installation takes a few seconds to complete.
+    </Message>
 
-    <Lightbox src="scaleway-nextcloud_database.webp" alt="" />
-2. Open a web browser and enter: `http://YOUR.CLOUD.SERVER.IP/nextcloud` (For example if your Instance's IP is `212.47.0.0` the input must be: `http://212.47.0.0/nextcloud`).
+## Log in to NextCloud
 
-    The installation script prompts you to include a username and password as well as the database credentials.
-3. Fill in the required information and press **Enter** to finalize the installation:
-    <Lightbox src="scaleway-nextcloud_setup.webp" alt="" />
+Once the installation completes, you are redirected to the login page. Enter the username and password you set previously and press **Enter** on your keyboard.
 
-<Message type="note">
-  The installation takes a few seconds to complete. 
+<Message type="tip">
+  After completing the NextcCloud installation, your browser may navigate to an incorrect URL. A quick workaround is to click the "Back" button in the browser to access the NextCloud login page.
 </Message>
 
-## Log in to Nextcloud
+You are now connected to your personal cloud storage and can start using it.
 
-Once the installation completes, you are redirected to the login page.
+<Lightbox src="scaleway-instantapp-nextcloud-splash.webp" alt="" />
 
-Enter the username and password you set previously and press **Enter** on your keyboard:
+## Opening the mail ports to send emails
 
-<Lightbox src="scaleway-nextcloud_login.webp" alt="" />
+ <Message type="note">
+  In order to prevent spam, our infrastructure does not allow access to remote mail ports by default. However, since NextCloud requires the ability to send emails, you have to enable access to these ports. Your account needs to undergo a validation proces before you can enable the SMTP ports. [Learn more about identiy verification](/console/my-account/how-to/verify-identity).
```

Comment:
```suggestion
  In order to prevent spam, our infrastructure does not allow access to remote mail ports by default. However, NextCloud requires the ability to send emails, so you must enable access to these ports. Your account needs to undergo a validation process before you can enable the SMTP ports. [Learn more about identity verification](/console/my-account/how-to/verify-identity).
```